python/lsst/afw/cameraGeom/testUtils.py

```diff
@@ -104,6 +106,9 @@ def __init__(self,
             CameraSys(TAN_PIXELS, self.name): pixelToTanPixel,
             CameraSys(ACTUAL_PIXELS, self.name): afwGeom.RadialXYTransform([0, 0.95, 0.01]),
         }
+        if crosstalk is None:
+            crosstalk = [[0.0 for _ in range(numAmps)] for _ in range(numAmps)]
```

Comment:
Shouldn't this be `[0,0]` (or maybe `[]`) based on the logic in `afw/cameraGeom/Detector.h:136`? Otherwise, it looks like there is a crosstalk matrix that just happens to be all zero.